### PR TITLE
Spelling and example syntax update

### PR DESCRIPTION
**Pull request by dglinder**
Per current documentation (https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#version-comparison) the "version_compare" test has been renamed to "version" as of Ansible 2.5.  Also minor updates to spelling, an update of the example to use the current "is" nomenclature, and fix test to "==".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Example Playbook
         - role: rhel7-stig
           when:
             - ansible_os_family == 'RedHat'
-            - ansible_distribution_major_version | version_compare('7', '=')
+            - ansible_distribution_major_version is version('7', '==')
```
Comment (jamescassell) [inline README.md]: No. The alias is not deprecated, and version doesn't work on older ansible versions.

File: README.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ Role Variables
 | `rhel_07_######` | [see defaults/main.yml](./defaults/main.yml)  | Individual variables to enable/disable each STIG ID. |
 | `rhel7stig_gui` | `no` | Whether or not to run tasks related to auditing/patching the desktop environment |
 | `rhel7stig_system_is_router` | `no` | Run tasks that disable router functions. |
-| `rhel7stig_antivirus_required` | `no` | Run tasks related to Anit-Virus package installation. |
```
Comment (jamescassell) [inline README.md]: Fine.

File: README.md (hunk 1 above)
Comment (dglinder) [reply README.md]: Understood - I'll revert to version_compare and resubmit the PR.

**Comment by dglinder** [comment]
Reverted to previous version check test.

**Comment by dglinder** [comment]
Ready to re-review and commit

**Pull request merged**